### PR TITLE
Fix RNA BAM path

### PR DESCRIPTION
**Pull request by tavinathanson**
A test would be a good idea, but not sure I need to block on that?

cc @arahuja

### CODE REVIEW

File: cohorts/load.py [hunk 1]
```diff
@@ -537,7 +537,7 @@ def _load_single_patient_neoantigens(self, patient, variant_type,
             if df_isovar is None:
                 import logging
                 logging.disable(logging.INFO)
-                rna_bam_file = AlignmentFile(patient.rna_bam.path)
+                rna_bam_file = AlignmentFile(patient.tumor_sample.bam_path_rna)
```
Comment (arahuja) [inline cohorts/load.py:544]: `tumor_sample` could be `None` worth checking? Not sure what to do in that case.

**Comment by arahuja** [comment]
👍

File: cohorts/load.py (hunk 1 above)
Comment (arahuja) [inline cohorts/load.py:544]: As could `bam_path_rna` which seems more common

Comment (tavinathanson) [inline cohorts/load.py:544]: Just added those checks 👍

**Pull request merged**